src/WishlistView/reducer.js

```diff
@@ -0,0 +1,60 @@
+import {
+    MARK_PURCHASED
+} from './actions';
+
+// TODO: fetch initial state
+let index = 0;
```

Comment:
mock data is useful but defining it yourself is a major pain in the ass. Check out this package https://github.com/Klowner/casual-browserify which will generate mocks for you.